Bottom: fb9ff9583e92dad1ee0a91d98d5e0a4cf91d9175
Top:    80a0d37cffe301b96a279c0e884652f977810986
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-07-24 17:05:16 +0100

Refresh of shiny-3d-eye-candy-0

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index b73e783..a9e7795 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -770,7 +770,6 @@ ghid_show_crosshair (gboolean paint_new_location)
   int vcd = VCD * gport->zoom;
   static int done_once = 0;
   static GdkColor cross_color;
-  extern float global_depth;
 
   if (!paint_new_location)
     return;
